docs/working_with_page.md

```diff
@@ -0,0 +1,226 @@
+!!! enterprise
+
+    # Working with a Page
```

Comment:
Page title should be outside the "enterprise" box (it doesn't make a difference for rendering, but that's the convention we keep in dev-doc)